### DISCUSSION TITLE
How to use realsense camera to build TSDF or ESDF

### DISCUSSION THREAD

**Discussion by KKXiaoKang** [NVlabs/curobo#647] [Software Q&A] [answered]
@balakumar-s 
* Hello，author
*  I  would like to inquire about how to generate ESDF or TSDF using a realsense camera, as I am now going to introduce my own defined static world obstacles or dynamically generate world obstacles like you did in the case collision_nvblox. yml
* if you have some examples to show, please tell me  thanks
* https://github.com/NVlabs/curobo/blob/main/examples/isaac_sim/collision_checker.py

**Comment by balakumar-s** [comment, marked as answer]
Does this help? https://nvlabs.github.io/curobo/latest/getting-started/volumetric_mapping.html

If you are using an older version of curobo, does this help? https://curobo.org/get_started/2d_nvblox_demo.html
